docs/source/tutorials/index.rst

```diff
@@ -15,4 +15,5 @@ Tutorials
    multilabel_classification
    token_classification
    pred_probs_cross_val
+   datalab
```

Comment:
TODO @huiwengoh : make this a pull-down with links to all datalab tutorials underneath the pulldown (starting with quickstart, then more advanced tutorial, then all of the reformatted existing tutorials).